files/build_templates/sonic_debian_extension.j2

```diff
@@ -111,6 +111,9 @@ sudo bash -c "echo enabled=false > $FILESYSTEM_ROOT/etc/sonic/updategraph.conf"
 # Copy SNMP configuration files
 sudo cp $IMAGE_CONFIGS/snmp/snmp.yml $FILESYSTEM_ROOT/etc/sonic/
 
+# Copy initial DHCP relay configuration file, will be overwritten after docker starts
```

Comment:
So until the DHCP servers are stored in the minigraph, this file must be edited to contain the actual DHCP server IPs?